pages/css/components/blankslate.md

```diff
@@ -77,18 +89,36 @@ Increases the size of the text in the blankslate
 
 ```html
 <div class="blankslate blankslate-large">
-  <h3>This is a blank slate</h3>
-  <p>Use it to provide information when no dynamic content exists.</p>
+  <img src="https://ghicons.github.com/assets/images/light/Pull%20Request.png" alt="Pull requests" class="mb-3">
+  <h3 class="mb-1">You don’t seem to have any pull requests.</h3>
+  <p>Pull requests help you discuss potential changes before your changes are merged into the base branch.</p>
+  <button class="btn btn-primary my-3" type="button">New pull request</button>
+  <p><button class="btn-link" type="button">Learn more</button></p>
 </div>
 ```
 
-##### No background
+##### No border
+
+To remove the border, use the [border utliity](/css/utilities/borders) `.border-0`.
+
+```html
+<div class="blankslate border-0">
+  <img src="https://ghicons.github.com/assets/images/light/Pull%20Request.png" alt="Pull requests" class="mb-3">
+  <h3 class="mb-1">You don’t seem to have any pull requests.</h3>
+  <p>Pull requests help you discuss potential changes before your changes are merged into the base branch.</p>
+  <button class="btn btn-primary my-3" type="button">New pull request</button>
+  <p><button class="btn-link" type="button">Learn more</button></p>
+</div>
+```
 
-Removes the `background-color` and `border`.
+Due to the update to the component, we will be deprecating the use of `.blankslate-clean-background` which removes the `background-color` and `border`.
```

Comment:
Can you clarify this a little bit? Does `the update of the component` refer to a design refresh?